services/venice-router/src/main/java/com/linkedin/venice/router/MetaDataHandler.java

```diff
@@ -320,7 +321,7 @@ private void handleLatestValueSchemaLookup(ChannelHandlerContext ctx, VenicePath
 
   private void handleUpdateSchemaLookup(ChannelHandlerContext ctx, VenicePathParserHelper helper) throws IOException {
     String storeName = helper.getResourceName();
-    checkResourceName(storeName, "/" + TYPE_VALUE_SCHEMA + "/${storeName}/${valueSchemaId}");
+    checkResourceName(storeName, "/" + TYPE_UPDATE_SCHEMA + "/${storeName}/${valueSchemaId}");
```

Comment:
`TYPE_UPDATE_SCHEMA` is not good name since people may think this endpoint is used to update schema.. 
Is it possible to rename it to be `TYPE_GET_UPDATE_SCHEMA` instead?

Reply:
Sure